### ISSUE TITLE
2.x libraries aren't available at the documented URL

### ISSUE THREAD

**Issue by mattdeco** [Adobe-Marketing-Cloud/video-heartbeat#10]
The Video Heartbeat Developer Guide was updated last week with documentation for the new 2.x version of the library. However, it doesn't appear that the 2.x version of the library has actually been published on GitHub or released in a way that would make it available for public use. The rest of this issue is in reference to the JavaScript SDK, but it looks like the iOS and Android SDKs are missing as well.

In the [Getting Started section of the guide](https://marketing.adobe.com/resources/help/en_US/sc/appmeasurement/hbvideo/js_2.0/r_vhl_getting-started-js.html), under the Download the SDK heading at the bottom of the page, there's a link to https://github.com/Adobe-Marketing-Cloud/video-heartbeat/releases and instructions to "Navigate to the tag which refers to latest library version for JavaScript". Unfortunately, the last version available as of filing this issue is 1.5.7, released 22 days ago (June 13th), while the 2.x Heartbeat SDK Developer Guide was published on June 30th. Additionally, there are [notes in the Developer Guide](https://marketing.adobe.com/resources/help/en_US/sc/appmeasurement/hbvideo/js_2.0/c_vhl_ch-logs_js.html) that indicate the 2.x version was released on July 1st.

I looked at the Adobe Marketing Cloud GitHub account and there is a separate repository for 2.x (https://github.com/Adobe-Marketing-Cloud/video-heartbeat-v2) but it doesn't appear to actually contain the 2.x libraries or fully-baked examples for players.

The 2.x library seems to include a much-improved API and better documentation that I would really like to take advantage of. It would be great if you could make them available for use.

To recap, here are some things I suggest to address this ticket:

- Make the Video Heartbeat 2.x SDKs available for public use
- Update the Video Heartbeat Developer Guide to link to the correct GitHub repository / location of the 2.x SDKs

Any help addressing this issue would be greatly appreciated!

**Comment by mattdeco** [comment]
The libraries have been added at https://github.com/Adobe-Marketing-Cloud/video-heartbeat-v2/tree/master/sdks -- I'll go ahead and close this issue.
